meson.build

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+project('eos-knowledge-renderer', 'c', version: '0.0.0', license: 'LGPL2.1+',
+    meson_version: '>= 0.42.0',
+    default_options: ['c_std=c99', 'warning_level=3'])
+
+gnome = import('gnome')
+pkg = import('pkgconfig')
+
+version_components = meson.project_version().split('.')
+api_version = version_components[0]
```

Comment:
You might consider not having an API version since this is a "private" library; although I'm not sure if I will regret suggesting that later.